mewarpx/mewarpx/diags_store/flux_diagnostic.py

```diff
@@ -797,6 +812,37 @@ def _check_charge_conservation(self):
                 % (mwxrun.get_it(), net_current)
             )
 
+    def _load_checkpoint_flux(self):
+        """Function to load the flux data from a simulation checkpoint during
+        a restart. The 'old' flux data will be recorded in the fullhist_dict so
+        that the simulation output will contain the full flux history and not
+        just the flux timeseries after the current restart.
+        """
+        # the current simulation step presumably matches the restart step
+        restart_step = mwxrun.get_it()
+
+        # get the flux diag file for the current step
+        flux_diag_file = os.path.join(
+            mwxrun.checkpoint_dir, self.FLUX_DIAG_DIR,
+            f"fluxdata_{restart_step:010}.dpkl"
+        )
+        # throw an error if flux diag file does not exist
+        if not os.path.isfile(flux_diag_file):
+            raise RuntimeError(
+                f"{flux_diag_file} does't exist but is needed to restart."
+            )
+
+        # use the current runinfo object to load the old flux data
+        # this assumes conductors and species are the same between runs
```

Comment:
```suggestion
        # Use the current runinfo object to load the old flux data.
        # This assumes conductors and species are the same between runs.
```
Normally I haven't been picky about comments grammar, but reading this as 2 sentences vs. one run-on sentence makes a big difference.